src/Plugin/Block/IslandoraAdvancedSearch.php

```diff
@@ -14,13 +18,35 @@
  *   admin_label = @Translation("Islandora advanced search"),
  * )
  */
-class IslandoraAdvancedSearch extends BlockBase {
+class IslandoraAdvancedSearch extends BlockBase implements ContainerFactoryPluginInterface {
```

Comment:
These search blocks seem to all need the same formbuilder access, should an intermediate block be made to extend or not worth the effort?